pkg/cloudprovider/aws/aws.go

```diff
@@ -128,10 +137,26 @@ type Volumes interface {
 	DeleteVolume(volumeName string) error
 }
 
+// InstanceGroups is an interface for managing cloud-managed instance groups / autoscaling instance groups
+// TODO: Allow other clouds to implement this
+type InstanceGroups interface {
+	// Set the size to the fixed size
+	Resize(instanceGroupName string, size int) error
```

Comment:
These methods would work better as "ResizeInstanceGroup", etc. Because the interface is implemented by AWSCloud, we end up with AWSCloud.Resize and AWSCloud.Describe, which could potentially cause confusion, especially when we pull this up into cloudprovider.